### PR TITLE
INV-36 Added filter information to GET endpoint on inventory item col…

### PR DESCRIPTION
**Pull request by jennshevach**
Added filter information to GET endpoint on inventory item collection.

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -70,11 +70,15 @@ Any response that does not provide a ```200 OK``` response code should be treate
 
 # Group Inventory Item
 
-## Inventory Item Collection [/accounts/{accountId}/inventoryitems{?page,pagesize,orderby}]
+## Inventory Item Collection [/accounts/{accountId}/inventoryitems{?q,isActive,manufacturerIds,partNumbers,page,pagesize,orderby}]
 This endpoint allows for the retrieval of a list of inventory items as well as the creation of new ones
 
 + Parameters
     + accountId: 2 (required, number) - {INT} ID of the account in the form of an integer
+    + q: Sample (string, optional) - Custom query object. Searches name and description fields.
```
Comment (jwholme2-dude) [inline apiary.apib]: Maybe instead of Sample use the actual query parameter, like manufacturerIds  ?

**Comment by jwholme2-dude** [comment]
+1

File: apiary.apib (hunk 1 above)
Comment (jennshevach) [reply apiary.apib]: Done.

**Comment by jay-scott** [comment]
+1

**Pull request merged**